pkg/apis/admission/types.go

```diff
@@ -63,6 +61,22 @@ type AdmissionReviewSpec struct {
 	UserInfo authentication.UserInfo
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AdmissionResponse describes an admission response.
+type AdmissionResponse struct {
+	metav1.TypeMeta
+
+	// Status is filled in by the webhook and indicates whether the admission
+	// request should be permitted.
+	// +optional
+	Status AdmissionReviewStatus
+
+	// The json-patch body. Must be in compliance with RFC 6902.
+	// +optional
+	PatchBody []byte
```

Comment:
I would call this just Patch.

Reply:
As discussed, we need a PatchType field or something.

(we decided that this was probably a little better than extending via doing a one-of arrangement with JSONPatch, StrategicMergePatch, etc.)